mds_loadtest/.gitignore

```diff
@@ -0,0 +1 @@
+*/loadTestConstants.py
```

Comment:
Can you rename the directory name to load_testing or something.
The prefix mds_ seems redundant